app/views/components/docs/heading.yml

```diff
@@ -0,0 +1,32 @@
+name: Heading
+description: A text heading
+body: |
+  A heading tag with an optional id attribute, used predominantly to the left of content on consultations and publications.
+
+  Real world examples:
+
+  - [Publication](/government/publications/recognising-the-terrorist-threat)
+  - [Consultation](/government/consultations/proposal-for-the-future-of-rotherham-goldthorpe-jobcentre)
```

Comment:
These links 404 - I don't think component guide is normally (or ever) on GOV.UK so would need fully qualified links

Reply:
Fixed